packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1099,7 +1100,7 @@ async function handler(
   const calEventUserFieldsResponses =
     "calEventUserFieldsResponses" in reqBody ? reqBody.calEventUserFieldsResponses : null;
 
-  const iCalUID = originalRescheduledBooking?.iCalUID ?? `${uid}@cal.com`;
+  const iCalUID = originalRescheduledBooking?.iCalUID ?? `${uid}@${ICAL_UID_DOMAIN}`;
```

Comment:
`suggestion`: I've seen this string being computed a few times throughout the PR, thoughts on abstracting it to a function w/ documentation so it can be easily reused and documented?

Reply:
@exception thoughts on this commit? f4a7377d283208f1c46519b7c14f786f0ef302e9

Reply:
Looks good!